### PR TITLE
ReplicatedPG: add runtime pg log query cli support

### PR DESCRIPTION
**Pull request by cxwshawn**
Signed-off-by: Xiaowei Chen <chen.xiaowei@h3c.com>

### CODE REVIEW

File: src/osd/ReplicatedPG.cc [hunk 1]
```diff
@@ -752,6 +752,13 @@ int ReplicatedPG::do_command(cmdmap_t cmdmap, ostream& ss,
     f->flush(odata);
     return 0;
   }
+  else if (command == "log") {
+    f->open_object_section("pg_log_t");
```
Comment (liewegas) [inline src/osd/ReplicatedPG.cc]: pg_log I think

**Comment by liewegas** [comment]
otherwise lgtm!

**Comment by cxwshawn** [comment]
@liewegas fixed, please review again.

**Comment by yuyuyu101** [comment]
I suggest add a test for this .

**Comment by athanatos** [comment]
This can go in, but it needs a test in one of the cli test scripts.  I'm closing for now, feel free to reopen with the test!

**Pull request closed without merging**